app/views/finders/_document.mustache

```diff
@@ -1,4 +1,5 @@
-<li class="document">
+<li class="document{{#document.top_result}} document--top{{/document.top_result}}">
+  <p>{{#document.top_result}}Most relevant result{{/document.top_result}}</p>
```

Comment:
```suggestion
  {{#document.top_result}}<p>Most relevant result</p>{{/document.top_result}}
```

Reply:
The reason why I am suggesting this is because if there is no top results, it will add an empty `<p>` tag